encodings/dict/benches/chunked_dict_array_builder.rs

```diff
@@ -43,13 +21,6 @@ const BENCH_ARGS: &[(usize, usize, usize)] = &[
     (8000, 10, 1000),
     (8000, 100, 1000),
     (8000, 1000, 1000),
-    // Too slow for CI
```

Comment:
I think it worth keeping these incase someone changes this in the future

Reply:
I'd really like to keep the runtime for micro-benchmarks reasonable. The current ones for this case are already very slow, even on my local machine. As a general guideline I think code should either be there or not.

Reply:
But they shouldn't be checked each time, I feel like this is comment to check to if altering this

Reply:
I'll keep it for now but we should revisit the parametrization tuple here, as it's extremely slow to bench.

Reply:
done

Reply:
We can comment out (8k, _, 1k)